### DISCUSSION TITLE
Add DMS as a suggestion in Microsoft's documentation

### DISCUSSION THREAD

**Discussion by symbiogenesis** [Mimetis/Dotmim.Sync#932] [Ideas]
I think DMS is fantastic, but if you are building a new application Microsoft's documentation directs you only to use Microsoft.Datasync

I think it would be worthwhile to send a PR to their documentation repo to explain the potential advantages of using DMS, and copy some of DMS's quick start instructions over to there.

https://learn.microsoft.com/en-us/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/offline?pivots=vs2022-windows

**Comment by Mimetis** [comment]
Yes, would be awesome ;)
